docs/guides/cloud/recorded-runs.mdx

```diff
@@ -317,6 +317,12 @@ canceled by members of the project.
 
   <DocsImage src="/img/dashboard/cancel-run-error.png" alt="cancel-run-error" />
 
+:::tip
+
+&#8239<Icon name="check-circle" color="green" /> See our [Run Cancellation](/guides/cloud/smart-orchestration/run-cancellation) guide to learn how to improve developer resolution time, reduce your monthly CI bill and free-up CI resources with `Auto Cancellation`.
```

Comment:
```suggestion
&#8239<Icon name="check-circle" color="green" /> See our [Run Cancellation](/guides/cloud/smart-orchestration/run-cancellation) guide to learn how to improve developer resolution time, reduce your monthly CI bill and free up CI resources with `Auto Cancellation`.
```

![image](https://github.com/cypress-io/cypress-documentation/assets/66998419/fa74fc9a-8df9-491a-a4d7-1095f6f37e41)

Reply:
Thanks Mike. I reused this from another area in the docs, so I can hit that too.